tests/h/services/search_index/service_test.py

```diff
@@ -154,7 +146,7 @@ def test_it(self, queue, search_index):
             "test_tag",
         )
 
-        queue.add_annotations_between_times.assert_called_once_with(
+        indexer.add_annotations_between_times.delay.assert_called_once_with(
```

Comment:
The test change corresponding to the real change made by this PR: scheduling `add_annotations_between_times()` asynchronously instead of calling it directly. Note the new module-scoped `indexer` fixture made this change easy to make

Reply:
You want to try the old websocket tests if you enjoy a challenge